### ISSUE TITLE
error in training AN4data

### ISSUE THREAD

**Issue by slbinilkumar** [SeanNaren/CTCNetworkClient#5]
I am getting this error while training the data .

lua: ...enspark/torch/install/share/lua/5.1/nn/Container.lua:67: 
In 3 module of nn.Sequential:
...orch/install/share/lua/5.1/nn/BatchNormalization.lua:80: got 11-feature tensor, expected 400
stack traceback:
    [C]: in function 'assert'
    ...orch/install/share/lua/5.1/nn/BatchNormalization.lua:80: in function 'checkInputDim'
    ...orch/install/share/lua/5.1/nn/BatchNormalization.lua:102: in function <...orch/install/share/lua/5.1/nn/BatchNormalization.lua:101>
    (tail call): ?
    [C]: in function 'xpcall'
    ...enspark/torch/install/share/lua/5.1/nn/Container.lua:58: in function 'rethrowErrors'
    ...nspark/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function <...nspark/torch/install/share/lua/5.1/nn/Sequential.lua:41>
    (tail call): ?
    ./Network.lua:95: in function 'opfunc'
    ...e/censpark/torch/install/share/lua/5.1/optim/sgd.lua:44: in function 'sgd'
    ./Network.lua:112: in function 'trainNetwork'
    AN4CTCTrain.lua:40: in main chunk
    [C]: ?

WARNING: If you see a stack trace below, it doesn't point to the place where this error occured. Please use only the one above.
stack traceback:
    [C]: in function 'error'
    ...enspark/torch/install/share/lua/5.1/nn/Container.lua:67: in function 'rethrowErrors'
    ...nspark/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function <...nspark/torch/install/share/lua/5.1/nn/Sequential.lua:41>
    (tail call): ?
    ./Network.lua:95: in function 'opfunc'
    ...e/censpark/torch/install/share/lua/5.1/optim/sgd.lua:44: in function 'sgd'
    ./Network.lua:112: in function 'trainNetwork'
    AN4CTCTrain.lua:40: in main chunk
    [C]: ?

**Comment by SeanNaren** [comment]
Hmm this is very strange have you made any changes to any other files? Downloaded the an4 dataset and ran the bash script to convert them to WAV, then change the path correctly in AN4CTCTrain?

**Comment by slbinilkumar** [comment]
yes i had done the same you had mentioned. But i had faced an issue in installation of Audio Library for Torch
shared library error 
 sudo apt-get install libfftw3-dev . Then i had installed it from the source,and it gone fine and run the code .
